vg/vgtex/canvas.go

```diff
@@ -0,0 +1,291 @@
+// Copyright 2016 The gonum Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package vgtex provides a vg.Canvas implementation for LaTeX, targeted at the TikZ/PGF LaTeX package:
+//  https://sourceforge.net/projects/pgf
+//
+// vgtex generates PGF instructions that will be interpreted and rendered by LaTeX.
+// vgtex allows to put any valid LaTeX notation inside plot's strings.
+package vgtex
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"image/color"
+	"io"
+	"math"
+	"strings"
+
+	"github.com/gonum/plot/vg"
+)
+
+const degPerRadian = 180 / math.Pi
+
+const (
+	defaultHeader = `%%%%%% generated by gonum/plot %%%%%%
+\documentclass{standalone}
+\usepackage{pgf}
+\begin{document}
+`
+	defaultFooter = "\\end{document}\n"
+)
+
+// Canvas implements the vg.Canvas interface, translating gonum/plot
+// drawing primitives into PGF ones.
```

Comment:
"Canvas implements the vg.Canvas interface, translating drawing primitives from gonum/plot to PGF."